OrbitBase/ThreadPool.cpp

```diff
@@ -19,6 +19,7 @@ class ThreadPoolImpl : public ThreadPool {
                           absl::Duration thread_ttl);
 
   size_t GetPoolSize() override;
+  size_t GetNBusyThreads() override;
```

Comment:
GetNumberOfBusyThreads/GetBusyTheadCount, I prefer the first one or some even better name I have not though about.

Reply:
Oh yes that's definitely a better name, will update it, thanks!

Reply:
Done, thanks!